datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/AddOwnerModal.tsx

```diff
@@ -39,9 +40,9 @@ type SelectedActor = {
     urn: string;
 };
 
-export const AddOwnerModal = ({ visible, onClose, refetch }: Props) => {
+export const AddOwnerModal = ({ visible, onClose, refetch, urn }: Props) => {
     const entityRegistry = useEntityRegistry();
-    const { urn, entityType } = useEntityData();
+    const { entityType } = useEntityData();
```

Comment:
entity type needs to be provided as well since it may be a Tag and in other cases will be the main entity of the page

Reply:
Fixed and committed.
[https://github.com/linkedin/datahub/pull/4132/commits/f314d27106cd3135e7def6ac9db381a764eb4d73](url)